rust/blockstore/src/arrow/block/types.rs

```diff
@@ -130,8 +130,8 @@ impl Block {
             // fewer branches and instructions than if/else or matching on
             // boolean.
             // This is x86 asm for u8: https://rust.godbolt.org/z/698eYffTx.
-            left = if predicate == true { mid + 1 } else { left };
-            right = if predicate == false { mid } else { right };
+            left = if predicate { mid + 1 } else { left };
```

Comment:
While I am sure this does not matter, I believe this is taken from std lib. Is std lib not clipped?

Reply:
It's often excluded from clippy as new lints are added.